tox.ini

```diff
@@ -22,18 +22,22 @@ commands =
     pip install -e .[test]
     pytest --cov=hibpcli --cov=tests --cov-report term-missing  --cov-report html
 
-[testenv:flake8]
-deps =
-    flake8
-    flake8-click
-    flake8-bugbear
-commands =
-    flake8 src tests setup.py
+[testenv:flake8]	
+deps =	
+    flake8	
+    flake8-click	
+    flake8-bugbear	
+commands =	
+    flake8 src tests setup.py	
 
-[flake8]
-max-line-length = 88
+[flake8]	
+max-line-length = 88	
 ignore = E231  # clashes with black
 
+[testenv:pre-commit]
+skip_install = true
+deps = pre-commit
+commands = pre-commit run --all-files --show-diff-on-faliure
 
```

Comment:
This looks almost good - but this also tells me you did not run it :-)

There is a typo in failure which results in a `pre-commit: error: unrecognized arguments: --show-diff-on-faliure`

Ideally, you should have `tox` installed on your system to be able to execute both the tests and the linter.

If you are interested in how to setup `tox`, you could have a look at this long issue https://github.com/morepath/morepath_cerebral_todomvc/issues/2 or go to `tox`s documentation:
https://tox.readthedocs.io/en/latest/install.html

I can support you - but that is up to you :-)

Reply:
Fixed the type!